images/hail-gcp-notebook/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM gcr.io/deeplearning-platform-release/base-cpu
+
+RUN apt-get update && \
+    apt-get install -y openjdk-8-jre-headless g++ python3-pip libopenblas-base liblapack3 && \
```

Hadolint is suggesting `--no-install-recommends`:

```suggestion
    apt-get install -y --no-install-recommends openjdk-8-jre-headless g++ python3-pip libopenblas-base liblapack3 && \
```